wao.io.siteoptimizationsubdomainstage.json

```diff
@@ -0,0 +1,21 @@
+{
+  "providerId": "wao.io",
+  "providerName": "wao.io",
+  "serviceId": "SiteOptimizationSubDomainStage",
+  "serviceName": "Site Optimization Sub Domain Stage",
+  "version": 1,
+  "description": "Enables wao.io site optimization for a sub domain.",
+  "multiInstance": true,
```

Comment:
Why is this multiInstance?  MultiInstance only impacts TXT records (you have none).  Suggest you remove this.

Reply:
I was under the impression that MultiInstance also works with CNAME records.

Reply:
We then will have to check whether it will be possible to use multiple templates with host variables.

Reply:
Hi, multiInstance is there if multiple templates need to exist on the same host. With CNAME it has no chance to work.
For different hosts multiInstance is not needed.

Reply:
Btw you want to replace %host% with @ as host is added to the template instance automatically.